R/model_info.R

```diff
@@ -52,7 +52,17 @@ model_info.lm <- function(model, ...) {
 #' @rdname model_info
 #' @export
 model_info.randomForest <- function(model, ...) {
-  type <- model$type
+  if (model$type == "classification" & !is.null(list(...)$task_subtype)) {
+    if (list(...)$task_subtype){
+      type <- "multiclass classification"
```

Comment:
lets keep it regression / classification / multiclass

Reply:
Done